src/engine/Module.v3

```diff
@@ -170,11 +170,11 @@ class FuncDecl(sig_index: int) extends Decl {
 	def findExHandler(instance: Instance, tag: Tag, throw_pc: int) -> ExHandler {
 		return findHandler("findExHandler", handlers, instance, tag, throw_pc);
 	}
-	def findSuspendHandler(instance: Instance, tag: Tag, resume_pc: int) -> ExHandler {
-		return findHandler("SusHandler", suspend_handlers, instance, tag, resume_pc);
-	}
-	def findSwitchHandler(instance: Instance, tag: Tag, resume_pc: int) -> ExHandler {
-		return findHandler("switch", switch_handlers, instance, tag, resume_pc);
+	def findSuspensionHandler(kind: SuspensionType, instance: Instance, tag: Tag, resume_pc: int) -> ExHandler {
```

Comment:
AFAICT the suspension type is only used here to select a string for tracing. If we have tracing further up in callers (e.g. a line of output to indicate whether a suspend or a switch is happening), then we can simplify this a bit.